data_filter_example/data_filter_example/opa.py

```diff
@@ -185,18 +196,19 @@ class queryTranslator(object):
 
     # Maps supported Rego relational operators to SQL relational operators.
     _sql_relation_operators = {
-        'eq': '=',
-        'equal': '=',
-        'neq': '!=',
-        'lt': '<',
-        'gt': '>',
-        'lte': '<=',
-        'gte': '>=',
+        "internal.member_2": "in",
```

Comment:
Wow, that has made it into the language _soo long ago_. I'm glad this is finally fixed.